packs/JS.json

```diff
@@ -0,0 +1,450 @@
+{
+   "cgdbId":null,
+   "code":"JS",
+   "name":"Jade Sea",
+   "releaseDate":"2021-02-15",
+   "cards":[
+      {
+         "code":"19001",
+         "type":"location",
+         "name":"Wraith",
+         "octgnId": null,
+         "quantity":3,
+         "unique":true,
+         "faction":"baratheon",
+         "loyal":false,
+         "cost":0,
+         "traits":[
+            "Warship"
+         ],
+         "text":"While you control Dale Seaworth, he gains stealth.\n<b>Action:</b> Kneel and sacrifice Wraith to choose and kneel a location.",
+         "deckLimit":3,
+         "illustrator":"Luciano Neves",
+         "imageUrl": "https://throneteki.ams3.cdn.digitaloceanspaces.com/packs/FH/1_Wraith_ENG.png"
```

Comment:
These should all be `JS` instead of `FH`, e.g. https://throneteki.ams3.cdn.digitaloceanspaces.com/packs/JS/1_Wraith_ENG.png

Reply:
Thanks, good catch. Fixed.